x/globalfee/feechecker/utils.go

```diff
@@ -122,21 +94,19 @@ func checkExecMsgReportFromReporter(ctx sdk.Context, oracleKeeper *oraclekeeper.
 	// If cannot get message, then pretend as non-free transaction
 	msgs, err := me.GetMessages()
 	if err != nil {
-		return false, err
+		return false, nil
```

Comment:
why not return err na

Reply:
it's the old logic. It's like it's just not our valid report message but it's still a valid message. so, we don't want to kick this message out.